bin/install.sh

```diff
@@ -99,6 +100,17 @@ function install_telemetry() {
     kubectl rollout status  deployment prometheus -n istio-telemetry --timeout=$WAIT_TIMEOUT
 }
 
+# Install Istio CNI
+function install_cni() {
+    step "cni.."
+    ISTIO_CNI_ARGS=
+    if [[ "${ISTIO_CLUSTER_ISGKE}" == "true" ]]; then
+        ISTIO_CNI_ARGS="--set cniBinDir=/home/kubernetes/bin"
+    fi
+    bin/iop istio-cni istio-cni $IBASE/istio-cni/ ${ISTIO_CNI_ARGS}
```

Comment:
Should we place CNI in istio-system, since it has the same level of security impact as Citadel ? 
May also be useful if in future we merge the node agents. But can be done in separate PR.